scripts/run-integration-tests.sh

```diff
@@ -0,0 +1,74 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+DIR=$(cd "$(dirname "$0")"; pwd)
+source $DIR/up-test-cluster.sh
+source $DIR/down-test-cluster.sh
```

Comment:
femto-nit: you can just have a single `scripts/lib/cluster.sh` script that contains these functions.

Reply:
Femto-Done